.github/workflows/conda-build.yml

```
@@ -14,50 +16,15 @@ jobs:
   build:
     name: ${{ matrix.CONFIG }}
     runs-on: ${{ matrix.runs_on }}
-    timeout-minutes: 360
+    timeout-minutes: 2160
```

I'd suggest a 9 hour timeout, given that the previous run for CUDA 11.8 / Python 3.12 was just a bit over 8 hours.

```suggestion
    timeout-minutes: 540
```